### ISSUE TITLE
unresolvable (REV PTR) clients not shown in web interface on version 5

### ISSUE THREAD

**Issue by rdslw** [pi-hole/pi-hole#3587]
**In raising this issue, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

3

---
**Expected behavior:**

Web Interface is showing clients which do NOT reverse resolve to name (e.g. 192.168.1.12) and show them as IPs.

It was working OK in pihole 4.x

**Actual behavior:**

Since version 5 (or 5.1 as I upgraded directly from 4.x to 5.1) if:
1. you use CONDITIONAL_FORWARDING
2. pihole is NOT dhcp server
3. some of the clients resolve to names and some do NOT

Pihole webinterface is NOT showing any client (and not counting in stats) which is IP only (meaning dns query send from pihole to my dhcp/dns server does not return name which is proper behaviour).

FTL works ok (answers requests from client).

**Steps to reproduce:**

as described above.

**Troubleshooting undertaken, and/or other relevant information:**

Checked /var/log/pihole.log and queries from such clients reach pihole, and pihole answers properly to them (some requests are blocked, some not, as usual).

**Comment by yubiuser** [comment]
Have you cleared your browser's cache after you upgraded your pihole?

**Comment by rdslw** [comment]
Yes, before clearing cache, dashboard stats was completly not loading, so I'm sure I did this.

Now dashboard and web interface works ok with one single difference: no rev-resolvable clients are not shown/nor counted in stats (e.g. % of blocked dropped from 28% to 2% :) )

p.s. Other clients are shown ok.
p.p.s. I just checked and /etc/pihole/pihole-FTL.db has relevent entries (from non resolvable clients) in queries table, so this may be bug in web interface code.
